RoddyCore/src/de/dkfz/roddy/ExitReasons.groovy

```diff
@@ -25,9 +25,11 @@ enum ExitReasons {
     analysisNotLoadable(244, "Could not load analysis"),
     severeConfigurationErrors(243, "Severe configuration errors occurred."),
     unhandledException(242, "Unhandled exception."),
-    unknownSSHHost(241, "Unknown SSH host."),
+    unknownSSHHost(241, "Could not setup SSH access with your configuration: The specified host is not available ."),
```

Comment:
Why are you substituting a short and clear message by a wordy one? Maybe the meaning should be 

"SSH remote host could not be reached"

or so?

Reply:
Yes, and that is really good. But in this particular case the message did not include any advise how to solve the problem. Also this problem is so complex (proxy, firewall, network config, dead remote, routing), that most advises would aim too short.